src/components/App/App.tsx

```diff
@@ -1252,6 +1264,9 @@ export default class App extends React.Component<AppProps, AppState> {
   };
 
   render() {
+    if (this.state.kicked) {
```

Comment:
I don't think we need state for this or the redirect component? We could just do a window.location = ?